src/main/kotlin/org/ionproject/integration/model/internal/timetable/TimetableTeachers.kt

```diff
@@ -0,0 +1,6 @@
+package org.ionproject.integration.model.internal.timetable
+
+data class TimetableTeachers(
+    var timetable: List<Timetable> = listOf(),
+    var teachers: List<CourseTeacher> = listOf()
```

Comment:
Same as Course

Reply:
They can't because they aren't filled all at the same time